vnext/ReactUWP/Views/KeyboardEventHandler.cpp

```diff
@@ -230,7 +226,7 @@ bool HandledKeyboardEventHandler::ShouldMarkKeyboardHandled(
     std::vector<HandledKeyboardEvent> const &handledEvents,
     HandledKeyboardEvent currentEvent) {
   for (auto const &event : handledEvents) {
-    if (event.key == currentEvent.key &&
+    if (event.key == toUpperOutOfPlace(currentEvent.key) &&
```

Comment:
>toUpperOutOfPlace [](start = 21, length = 17)

Please rebase master, we use code and key is removed.
also you can remove toUpperOutOfPlace